### ISSUE TITLE
Generate a normalized Charges database to populate model

### ISSUE THREAD

**Issue by bepetersn** [sc3/cookcountyjail#352]

**Comment by bepetersn** [comment]
It never occurred to me to ask what format would be most helpful.

I have done this in the way I first thought to do it. That is, I created a json file containing a list of unique citations, minimally normalized in a few of the most obvious ways to reduce duplicates. The idea would be that Wil could feed this json to his process route.

Is this sufficient? I have thought of a lot of different ways to do this.

**Comment by bepetersn** [comment]
This has been done. But more importantly, we need to port the parsing that v1 does to our new ORM

**Comment by bepetersn** [comment]
I have been working with charges a lot, and I did mostly get this to the point where I was generating a data migration in a script to initialize our new database. 

However, @wilbertom had convinced me to try to POST this directly to the v2 API to start conforming to our new standard. I thus stepped back from trying to generate this database migration, and tried to POST directly. It was that attempt that I finished.

Upon reflection, though, I still would like to generate a one-off data migration. Thus, I'm going to work on finishing this. One thing that needs to be added is generating initial charge descriptions as well as citations.
